### PR TITLE
 Fixing GetMessages Function

### PR DESCRIPTION
**Pull request by TcgVanguardTroll**
This Pull request fixes the getMessages method so that it fethces and iterets throguh the list of comments and the comment respectively

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,8 +5,6 @@
     <title>Jordan's Site</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-
-    <script src = "https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.js"></script>
```
Comment (tcwanggoogle) [inline portfolio/src/main/webapp/index.html:9]: Remove this by accident, or does it have some impact on your webpage?

**Review by tcwanggoogle — APPROVED** [review]
Looking forward to your new webpage!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ function addRandomFact(){
 }
 
 function getMessages(){
-    fetch('/data').then(response => response.json()).then((comment) => {
+    fetch('/data').then(response => response.json()).then((comments) => {
```
Comment (kmh287) [inline portfolio/src/main/webapp/script.js:86]: indentation looks off here. The continuation from the promise should be +2 indented from the `fetch`

**Review by kmh287 — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/index.html:9]: I was using a CDN in order to use Node.js functions but I have just requested access to Node.js

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/script.js:86]: Fixed

**Pull request merged**